plugins/ua_config_standard.c

```diff
@@ -9,6 +9,7 @@
 #include "ua_securitypolicy_none.h"
 #include "ua_types.h"
 #include "ua_types_generated_handling.h"
+#include "ua_client_highlevel.h"
```

Comment:
Separate PR or this one? I'd really prefer if this one is merged soon, since the conflicts are getting out of hand.